presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the `http-server.https.excluded-cipher` config
+  property to match Jetty's default excluded cipher suites. Specifically, all
```

Comment:
Let's not discuss Jetty here, as that's an implementation detail. One way is to say *"... to exclude insecure ciphers"*, but saying *"insecure"* is not great. More along the lines of *"... to exclude ciphers that are no longer allowed in modern TLS standards"* but that also needs work. Thoughts?